shell/components/VeeValidation.vue

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+import Vue from 'vue';
+import ErrorMessage from '@components/ErrorMessage/ErrorMessage.vue';
+import { ValidationProvider } from 'vee-validate';
+
+interface Data {}
+
+export default Vue.extend<Data, any, any, any>({
```

Comment:
Could we name the component something more human like `RCValidation`, following @rak-phillip concept?
At this point we could also move it within pkg, no?